app/lib/fs/form_related_documents_downloader.rb

```diff
@@ -18,8 +18,8 @@ def download_related_document_by_type(type)
       related_document_path = XSD_PATH
       related_document_type = 'CLS_F_XSD_EDOC'
     when :xslt_html
-      related_document_path = xml_manifest.xpath('//manifest:file-entry[@media-destination="screen" or @media-destination="view"]')&.first['full-path']
-      related_document_path&.gsub!(/\\/, '/')
+      related_document_node = xml_manifest.at_xpath('//manifest:file-entry[@media-destination="x-xslt-ro"]')
+      related_document_path = related_document_node&.[]('full-path')&.gsub(/\\/, '/')
```

Comment:
toto je velmi zvlastny zapis. co sa tu snazime ziskat?

Reply:
upravil som to v https://github.com/slovensko-digital/govbox-pro/pull/723/commits/6dffe02fe31b41c0c5b0860d61ef1b0ae1ccf09b, nech je to ako predtým, len som upravil:
```diff
- &.first['full-path']
+ &.first&.[]('full-path')
```

keďže `xslt_html` nemusí existovať a potom by to vyhodilo chybu `undefined method '[]'`